test/e2e/service_test.go

```diff
@@ -36,6 +37,11 @@ func TestService(t *testing.T) {
 		test.serviceCreateDuplicate(t, "hello")
 	})
 
+	t.Run("create private service and change test change pf visibility", func(t *testing.T) {
+		test.serviceCreatePrivate(t, "hello-private")
+		test.serviceCreatePrivateUpdatePublic(t, "hello-private-public")
```

Comment:
do we need this?

`service update` should've done this simply after private service creation.

Reply:
I want to separate the concerns.